lib/chef/provider/cron.rb

```diff
@@ -237,7 +237,11 @@ def get_crontab_entry
         newcron = ""
         newcron << "# Chef Name: #{new_resource.name}\n"
         [ :mailto, :path, :shell, :home ].each do |v|
-          newcron << "#{v.to_s.upcase}=#{@new_resource.send(v)}\n" if @new_resource.send(v)
```

Comment:
Pretty much the entire file is using `@new_resource`, so I wanted to stay consistent. There's definite scope for a rewrite, but I wanted to fix this first.